packages/backend/src/open_payments/wallet_address/errors.ts

```diff
@@ -15,13 +16,16 @@ export const errorToCode: {
 } = {
   [WalletAddressError.InvalidUrl]: GraphQLErrorCode.BadUserInput,
   [WalletAddressError.UnknownAsset]: GraphQLErrorCode.BadUserInput,
-  [WalletAddressError.UnknownWalletAddress]: GraphQLErrorCode.NotFound
+  [WalletAddressError.UnknownWalletAddress]: GraphQLErrorCode.NotFound,
+  [WalletAddressError.DuplicateWalletAddress]: GraphQLErrorCode.Duplicate
 }
 
 export const errorToMessage: {
   [key in WalletAddressError]: string
 } = {
   [WalletAddressError.InvalidUrl]: 'invalid url',
   [WalletAddressError.UnknownAsset]: 'unknown asset',
-  [WalletAddressError.UnknownWalletAddress]: 'unknown wallet address'
+  [WalletAddressError.UnknownWalletAddress]: 'unknown wallet address',
+  [WalletAddressError.DuplicateWalletAddress]:
+    'wallet address is already registered'
```

Comment:
Final thing, let's call this 
```suggestion
    'Duplicate wallet address found with the same url'
```